Application/src/tracker/python/trex_sam3_interface.py

```
@@ -0,0 +1,1104 @@
+
+            text_scope = "session" if self._text_session_scope else f"frame {int(self._text_frame_index)}"
+            preview = self._render_preview_frame(
+                frame,
+                entry.output,
+                alpha=0.5,
+                text_prompt=self._text_prompt,
+                text_scope=text_scope,
+                box_payload=box_payload,
+            )
+            if TRex is not None:
+                TRex.imshow(f"SAM3 Preview", preview)
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)